airbyte-integrations/connectors/source-dremio/source_dremio/dremio.yaml

```diff
@@ -0,0 +1,55 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "{{ config['base_url']}}/api/v3"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "_dremio{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  catalogs_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "catalogs"
+      path: "/catalog"
+      primary_key: "id"
```

Comment:
primary_key should be defined outside the `options` block, otherwise normalization may fail